packages/jazz-tools/src/tools/coValues/account.ts

```diff
@@ -93,119 +93,40 @@ export class Account extends CoValueBase implements CoValue {
-    if (this.isLocalNodeOwner) return this;
-
-    const agent = this._raw.core.node.getCurrentAgent();
-
-    if (agent instanceof ControlledAccount) {
-      return coValuesCache.get(agent.account, () =>
-        Account.fromRaw(agent.account),
-      );
-    }
-
-    return new AnonymousJazzAgent(this._raw.core.node);
-  }
-
   declare profile: Profile | null;
   declare root: CoMap | null;
 
-  getDescriptor(key: string) {
-    if (key === "profile") {
-      return this._schema.profile;
-    } else if (key === "root") {
-      return this._schema.root;
-    }
-
-    return undefined;
-  }
-
-  get _refs(): {
-    profile: RefIfCoValue<Profile> | undefined;
-    root: RefIfCoValue<CoMap> | undefined;
-  } {
-    const profileID = this._raw.get("profile") as unknown as
-      | ID<NonNullable<this["profile"]>>
-      | undefined;
-    const rootID = this._raw.get("root") as unknown as
-      | ID<NonNullable<this["root"]>>
-      | undefined;
-
-    return {
-      profile: profileID
-        ? (new Ref(
-            profileID,
-            this._loadedAs,
-            this._schema.profile as RefEncoded<
-              NonNullable<this["profile"]> & CoValue
-            >,
-            this,
-            // eslint-disable-next-line @typescript-eslint/no-explicit-any
-          ) as any as RefIfCoValue<this["profile"]>)
-        : undefined,
-      root: rootID
-        ? (new Ref(
-            rootID,
-            this._loadedAs,
-            this._schema.root as RefEncoded<
-              NonNullable<this["root"]> & CoValue
-            >,
-            this,
-            // eslint-disable-next-line @typescript-eslint/no-explicit-any
-          ) as any as RefIfCoValue<this["root"]>)
-        : undefined,
-    };
-  }
-
-  /**
-   * Whether this account is the currently active account.
-   */
-  get isMe() {
-    return activeAccountContext.get().id === this.id;
-  }
-
-  /**
-   * Whether this account is the owner of the local node.
-   */
-  isLocalNodeOwner: boolean;
-  sessionID: SessionID | undefined;
-
   constructor(options: { fromRaw: RawAccount }) {
     super();
     if (!("fromRaw" in options)) {
       throw new Error("Can only construct account from raw or with .create()");
     }
-    this.isLocalNodeOwner =
-      options.fromRaw.id == options.fromRaw.core.node.getCurrentAgent().id;
 
     Object.defineProperties(this, {
       id: {
         value: options.fromRaw.id,
         enumerable: false,
       },
-      _raw: { value: options.fromRaw, enumerable: false },
       _type: { value: "Account", enumerable: false },
+      $jazz: {
+        value: new AccountJazzApi(this, options.fromRaw),
+        enumerable: false,
+      },
     });
 
-    if (this.isLocalNodeOwner) {
-      this.sessionID = options.fromRaw.core.node.currentSessionID;
-    }
-
     return new Proxy(this, AccountAndGroupProxyHandler as ProxyHandler<this>);
   }
 
+  // TODO Q: does it make sense to move Group methods into `$jazz` as well?
```

Comment:
Given that Groups cannot be extended with user-defined properties, requiring people to prefix all operations with `$jazz` feels like unnecessary boilerplate. The same question applies to Accounts methods